mlir/utils/performance/perfRunner.py

```diff
@@ -472,7 +472,8 @@ def benchmarkExternal(cls, commandLine, paths: Paths, arch, envs=dict()):
             outs, errs = p1.communicate(timeout=300)
             if len(errs) > 0:
                 print("MIOpen benchmark produced errors: ", errs.decode('utf-8'))
-                raise OSError(errs.decode('utf-8'))
+                if p1.returncode != 0:
```

Comment:
This is happening:
```
Running MIOpen Benchmark:  conv -F 1 -f NCHW -I NCHW -O NCHW -n 256 -c 512 -H 7 -W 7 -k 512 -y 3 -x 3 -p 1 -q 1 -u 1 -v 1 -l 1 -j 1 -m conv -g 1 -t 1

MIOpen benchmark produced errors:  MIOpen(HIP): Warning [FindSolutionImpl] Invalid config loaded from Perf Db: ConvAsmImplicitGemmGTCDynamicFwdXdlopsNHWC: fwd,nhwc,fp32,0,1,128,128,16,32,32,2,1,1,2,2,0,0,1,0,0,1,4,2,1,1,4,1,64,1,4,2,1,1,4,1,64,0. Performance may degrade.
```
Before I started raising signals, it would just message and continue, but with a signal, it halts.  I don't know where the invalid configs come from, but to preserve existing behavior I check the return code to see if the process itself failed.